src/nl/blog/2025/01/omscholen-naar-front-end-developer.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Omscholen naar Front-End Developer - Tips van een ervaringsdeskundige
+date: 2025-01-03 # jjjj - mm - dd
+author: Annemiek Nieboer
+summary: In dit blog deel ik mijn ervaringen als omscholer. Ik geef tips hoe je erachter komt of het front-end development vak wat voor voor je is, hoe je je omschoolt en hoe je aan je eerste baan als front-ender komt. 
+categories:
+  - Adventskalender
+---
+
+Ik vond het enorm spannend om te beginnen aan mijn omscholing van marketeer naar front-end developer. Een omscholing starten is één, maar ook echt aan de slag gaan als front-end developer is toch wel even andere koek. In dit blog deel ik mijn eigen inzichten en geef ik tips over de weg naar de keuze toe voor de omscholing, de omscholing zelf en hoe je vervolgens een baan vindt.
+
+### Omscholen naar front-end development: waar begin je?
```

Comment:
Dit worden `<h3>`s, terwijl `<h2>` beter zijn omdat je dan niet een niveau overslaat. (De blogposttitel wordt al een `<h1>`)

```suggestion
## Omscholen naar front-end development: waar begin je?
```

Reply:
Omy... voel mij even uber noob nu haha, werk eigenlijk nauwelijks met md bestanden en totaal niet bij stilgestaan dat dit h1/2/3's worden. Weer wat geleerd, thanks! Aangepast in 253bca2ecea5d0495da88c1e8d223f7b34356dd4